src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -28,9 +28,10 @@ struct event_entry_t {
   uint32_t time_to_live = DEFAULT_GLOBAL_VALUE;
   uint32_t max_retries = DEFAULT_GLOBAL_VALUE;
   uint32_t retry_sleep_duration = DEFAULT_GLOBAL_VALUE;
+  bool needs_migration = false;
   
   void encode(bufferlist& bl) const {
-    ENCODE_START(3, 1, bl);
+    ENCODE_START(4, 1, bl);
```

Comment:
if we changed from "2" to "3" post "reef", you don't need to change to "4" - you can keep at "3".
there are no compatibility requirements within the same version (both changes are in "squid").

Reply:
Wasn't aware that it is tied to the ceph versions
Thanks for informing me :)